sim/visuals/microbit.ts

```diff
@@ -822,7 +883,7 @@ namespace pxsim.visuals {
         private buttonsOuter: SVGElement[];
         private pins: SVGElement[];
         private pinGradients: SVGLinearGradientElement[];
-        private pinTexts: SVGTextElement[];
+        private pinTexts:{ [key: number]: SVGTextElement };
```

Comment:
@pelikhan is this looking better to you?

Reply:
that works